packages/fxa-auth-server/lib/payments/stripe-formatter.ts

```diff
@@ -46,3 +46,20 @@ export function stripeInvoiceToInvoicePreviewDTO(
   }
   return invoicePreview;
 }
+
+/**
+ * Formats an array of Stripe Invoice to the SubsequentInvoicePreview DTO format.
+ */
+export function stripeInvoicesToSubsequentInvoicePreviewsDTO(
+  invoices: Stripe.Invoice[]
+): invoiceDTO.SubsequentInvoicePreview[] {
+  const invoiceNext: invoiceDTO.subsequentInvoicePreviewsSchema = invoices.map(
```

Comment:
nit: should this be plural, invoicesNext?  It is a list.  Or, maybe we don't even name it, and just return the return value of the `.map` call without assigning it.

Reply:
Oh yes good idea. I'll just return the value of the map. Thank you.